inst/stan/chunks/debug.stan

```diff
@@ -1,17 +1,17 @@
     for (i in 1:refp_fnrow) {
       int j = 0;
       for (k in 1:dmax) {
-        j += is_nan(abs(pmfs[k, i])) ? 1 : 0;
-        j += is_inf(abs(pmfs[k, i])) ? 1 : 0;
+        j += is_nan(abs(ref_lh[k, i])) ? 1 : 0;
+        j += is_inf(abs(ref_lh[k, i])) ? 1 : 0;
```

Comment:
Replaced `pfms` with `ref_lh`. Could be `hazards` if we want to keep them in `transformed parameters`.  Btw, why do we use `abs` here (and `cmdstan` is asking to change it to `fabs`)?

Reply:
In 2.30.0 it warns about fabs. In 2.29.0 it warns about abs. I think the flip flopped on which to implement. I saw some issues to this effect but now can't find.

Reply:
Oh I didn't know they were switching around. My question was more about why we need the absolute here at all, why can't we just plug in the potentially negative values into `is_inf`? I am probably lacking some knowledge on `is_inf` here?

Reply:
uummmmm I don't know what past Sam was thinking here. Maybe lack support for negatives? I have no idea. Will test.